metadrive/utils/utils.py

```diff
@@ -31,6 +32,14 @@ def recursive_equal(data1, data2, need_assert=False):
     if isinstance(data2, Config):
         data2 = data2.get_dict()
 
+    if isinstance(data1, np.ndarray):
```

Comment:
I fix this error in the same way. Please merge main and solve conflict

Reply:
Great